physics/apsides.hpp

```diff
@@ -30,6 +34,20 @@ void ComputeApsides(Trajectory<Frame> const& reference,
                     DiscreteTrajectory<Frame>& apoapsides,
                     DiscreteTrajectory<Frame>& periapsides);
 
+// Computes a collision between a vessel and a rotating body.  |begin| and |end|
+// must be on opposite sides of the surface of the body (that is, a collision
+// must exist).  |radius| gives the radius of the celestial at a particular
+// position given by its latitude and longitude.
```

Comment:
Document the contract between `radius` and `reference_body.max_radius()` (do we use min_radius? should we?)